docs/Building.md

```diff
@@ -96,10 +96,6 @@ The full suite of system tests takes approximately 30 minutes to complete.  If y
 Docker, and are regularly running the system tests, a faster subset of system tests may be run which covers most
 mainline function.
 
-Both ST suites utilize Docker's [libnetwork network driver](https://github.com/docker/libnetwork), available in the
```

Comment:
I prefer not to have different changes in a single PR. I've just deleted this block in a different commit